recipe/bld.bat

```diff
@@ -21,11 +20,11 @@ cmake ^
 if %ERRORLEVEL% neq 0 exit 1
 
 :: Build step
-nmake VERBOSE=1
+cmake --build .  --target compiler-rt
```

Comment:
Could the build error come from the fact that we are building a specific target here? At least the logs suggest that the required `stats.cpp` (the only source of `clang_rt.stats`) source file is never compiled.

Reply:
Without the target, nothing at all gets compiled, see the CI of the previous commit, resp. https://github.com/llvm/llvm-project/issues/58893